content/zh-cn/nodes.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 节点
+status: Completed
+category: 概念
+---
+
+## 是什么
+
+节点是一台能与其他计算机（或节点）协同工作以完成一个共同任务的计算机。以你的笔记本电脑、调制解调器或打印机为例。它们都通过你的 wifi 网络进行通信和协作，各自代表一个节点。在[云计算](/zh-cn/cloud_computing/)中，节点可以是一台物理计算机，也可以是一台虚拟计算机（即虚拟机），甚至可以是[容器](/zh-cn/container/)。
```

Comment:
Recently, `cloud_computing.md` is renamed to `cloud-computing.md`. 😊
- https://github.com/cncf/glossary/blob/dev-zh/content/zh-cn/cloud-computing.md
- https://glossary.cncf.io/zh-cn/cloud-computing/

```suggestion
节点是一台能与其他计算机（或节点）协同工作以完成一个共同任务的计算机。以你的笔记本电脑、调制解调器或打印机为例。它们都通过你的 wifi 网络进行通信和协作，各自代表一个节点。在[云计算](/zh-cn/cloud-computing/)中，节点可以是一台物理计算机，也可以是一台虚拟计算机（即虚拟机），甚至可以是[容器](/zh-cn/container/)。
```